sportsreference/nfl/constants.py

```diff
@@ -352,7 +352,8 @@
     'punt_return_touchdowns': 'td[data-stat="punt_ret_td"]'
 }
 
-SEASON_PAGE_URL = 'http://www.pro-football-reference.com/years/%s.html'
+# SEASON_PAGE_URL = 'http://www.pro-football-reference.com/years/%s.html'
```

Comment:
Since this URL is no longer valid, we should probably get rid of this line entirely. I know it won't be used as it's a comment, but I think it would look cleaner if we just took it out altogether.

Reply:
Agreed!